### DISCUSSION TITLE
How do I get numbers only in the citations? Output (numerical format) Mine setup is using the Output (author-date format)

### DISCUSSION THREAD

**Discussion by ghost** [quarto-dev/quarto-cli#6256] [Q&A] [answered]
### Description

How do I get numbers only in the citations? 

Output (numerical format)

Mine setup is using the Output (author-date format)

What can I add to change this. I did not see it on the page https://quarto.org/docs/authoring/footnotes-and-citations.html

Thank You.

**Comment by mcanouil** [comment]
Choose the CSL that does that, see <https://quarto.org/docs/authoring/footnotes-and-citations.html#citation-style>.

PS: a good post title is short and concise.

**Reply by ghost** [reply]
I already read that page and it does not show to me what to place in the yaml to have the output as numeric.

**Reply by mcanouil** [reply]
You have to read the links in that section, go through all the styles, choose the CSL file and as described add it to your yaml.
<img width="1500" alt="Screenshot 2023-07-17 at 21 39 54" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/12b96bab-d6d3-4425-9a3e-60f50dd5ecaa">

Additionally, you have to the use the "right" syntax, as described <https://quarto.org/docs/authoring/footnotes-and-citations.html#sec-citations>.
The syntax output will depend on your CSL file.

**Reply by ghost** [reply]
I added to my _quarto.yml

format: 
     pdf:
         cite-method: biblatex

And so I get [1], [2], as output for the pdf, but not for html.

For html it still has author, date

Any ideas?

**Reply by ghost** [reply]
I am not asking about csl files, just wanna change the output format from author/date to numeric in html format. 

Is there something I can add to the yaml for html to make it work on the webpage also? 

(without searching through 8,500 files)

Thank You.

**Reply by mcanouil** [reply]
> (without searching through 8,500 files)

Start by providing a reproducible example to work with.
You only mentioned now your `_quarto.yml` which formats you are working with.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

The end.
````
`````

**Reply by ghost** [reply]
Wow you guys make it incredible hard to get help with this....

I just wanna change from author/date output to numeric output using the default citeproc that quarto uses. 

Of course, I would know what a minimal example is if Quarto would let use know in their documentation. 

It would also be helpful to not only know what a minimal working example is, but if Quarto had minimal working examples in their documentation, then the requesting of providing a minimal working example wouldn't sound so unhelpful. 

Thank You.

**Reply by mcanouil** [reply]
If you don't want to provide context, document or anything don't expect people to guess what you want.

I strongly believe I provided the answer to your question, until you give what I asked, there is nothing more I can do to help.

Following my own previous answer[^1]:

https://github.com/quarto-dev/quarto-cli/assets/8896044/58361ed3-4681-4216-bc71-5397d8d6cd49

[^1]: see <https://github.com/quarto-dev/quarto-cli/discussions/6256#discussioncomment-6471234>.

**Reply by JGobeil** [reply]
Hi, I think the problem comes from how the documentation is written.

I found this discussion trying to find what option to activate to have numbered citations. I already know, reading the documentation, that this is possible via a custom style. Still, because the "Output (numerical format)" is presented in the table as an example, this gives the impression that this is a result that you can get out of the box via a simple true/false flag or something similar. That leads people to look for an option that does not exist. 

I suggest that simply adding the information to the table will solve that misunderstanding. Something like "Output (default)" and Output(csl: diabetologia.csl, see Citation Style)

**Reply by mcanouil** [reply]
@JGobeil PR is welcome if you feel like contributing.

**Reply by JGobeil** [reply, marked as answer]
https://github.com/quarto-dev/quarto-web/pull/813

**Reply by mcanouil** [reply]
@JGobeil could you elaborate? (Possibly by email to <mickael.canouil@posit.co>)
(If you are referring to the thread itself, there is more to the story to what's in the actual thread).

Thanks for the PR!

**Reply by JGobeil** [reply]
@mcanouil Landing on this thread gave me a bad impression, that's all. If there is more to the story, I will believe you and will forget this initial impression :smile:

**Reply by cscheid** [reply]
> PS: You know that the way you interact is not really inviting collaborations?

Hi - I would like to ask you to review our [code of conduct](https://github.com/quarto-dev/quarto-cli/blob/main/.github/CODE_OF_CONDUCT.md). @mcanouil is simply responding to the fact that the best way to make a suggestion is, in fact, to make a PR. Please don't ascribe a tone to a message where there's a number of other alternative explanations (English is not mine nor @mcanouil's native language, to give the simplest possible explanations). Your message is not helpful, and we generally expect better. Specifically, your message falls under "[...] personal [...] attacks". Please strive to do better here in the future.

**Reply by JGobeil** [reply]
Sorry if I was misinterpreted, as English is not my first language either. I was not referencing the way he answered me but the tone of the previous conversation. @mcanouil answered me and contextualised the conversation, and I was quite happy by this. I will be more careful if I have to interact here in the future. 

p.s. I edited my message to remove the misformulated comment.
